pkg/auth/auth_token_handler.go

```diff
@@ -150,7 +150,7 @@ func (a *AuthTokenHandlerImpl) GetConfluentToken(mdsClient *mdsv1.APIClient, cre
 	if credentials.IsSSO {
 		if credentials.AuthRefreshToken != "" {
 			if authToken, err := refreshConfluentToken(mdsClient, credentials); err == nil {
-				return authToken, "", nil
+				return authToken, credentials.AuthRefreshToken, nil
```

Comment:
Interesting that we re-use the refresh token. Sounds like the refresh endpoint doesn't also refresh the refresh token?

Reply:
Nope.

It also doesn't return the current refresh token, which is why I originally assumed that it was one time use.

Reply:
I'm curious why that is. Let's merge since this is still an improvement!

Reply:
Sounds good. Just going to wait a bit on Monday to see if I can get some help from feature team to test this (since my manual testing setup doesn't work anymore...)